web_src/js/utils.test.js

```diff
@@ -133,17 +133,22 @@ test('toAbsoluteUrl', () => {
   expect(() => toAbsoluteUrl('path')).toThrowError('unsupported');
 });
 
-const uint8array = (s) => new TextEncoder().encode(s);
 test('encodeURLEncodedBase64, decodeURLEncodedBase64', () => {
+  // TextEncoder is Node.js API while Uint8Array is jsdom API and their outputs are not
+  // structurally comparable, so we convert to array to compare. The conversation
+  // can be removed once https://github.com/jsdom/jsdom/issues/2524 is resolved.
```

Comment:
The conversation can be removed?
Do you mean
```suggestion
  // structurally comparable, so we convert to array to compare. The conversion
  // can be removed once https://github.com/jsdom/jsdom/issues/2524 is resolved.
```
?